testing_exercise/testing.js

```diff
@@ -0,0 +1,47 @@
+function replaceWith(str, replaceThisChar, replacementChar) {
+	var arrayOfChars = str.split("");
+	arrayOfChars.forEach(function(val, index){
+		if (val === replaceThisChar) {
+			arrayOfChars[index] = replacementChar;
+		}
+	});
+	return arrayOfChars.join("");
+}
+
+function expand(arr, num) {
+	if (num === 1) {
```

Comment:
I was thinking that in the case of num === 1, or with no repeats, I would just return to save the work of going into the for loop, but I can see that it's just unnecessary code. I'll remove it.